pkg/daemon/ceph/osd/volume.go

```diff
@@ -56,6 +56,10 @@ var (
 	// The "ceph-volume raw" command is available since Ceph 14.2.8 as well as partition support in ceph-volume
 	cephVolumeRawModeMinCephVersion = cephver.CephVersion{Major: 14, Minor: 2, Extra: 8}
 
+	// The "ceph-volume lvm" version where things started to break batch mode
+	// See https://github.com/rook/rook/issues/6532 for more details
+	cephVolumeLVMDiskSortingCephVersion = cephver.CephVersion{Major: 14, Minor: 2, Extra: 13}
```

Comment:
We also need to check for 15.2.6 or newer, right? And not use the new flag for Octopus before .6?